lib/constants/navigation_constants.dart

```diff
@@ -0,0 +1,3 @@
+// Widget dimensions
+
+const double kAppNavBarHeight = 60.0;
```

Comment:
Learned that flutter recommends to add `k` as a prefix to global double/int constants. See this sample code as a guide: https://www.driftycode.com/flutter-global-constants-for-the-complete-project/

Reply:
Same comment as above